### ISSUE TITLE
3D View - bforartists_brush_panel addon - Add the Vertex Paint mode

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#2983]
The Bforartists_brush_panel addon can now start working with other brush modes. 

We will begin with the Vertex Paint mode

**Comment by Draise14** [comment]
Hopefully the broken vertex paint mode from Blender will fix this weekend merge, right now it's crashy, buggy and broken since the merged in the experimental vertex paint mode for sculpting and changed vertex colours to geometry nodes attributes (not sure which).

**Comment by ReinerBforartists** [comment]
That's development :)

**Comment by Draise14** [comment]
Ok, the issues are fixed with the latest devbuild 30-04-2022, meaning this can continue.

**Comment by Draise14** [comment]
Hey there @iyadahmed, I am going to tackle this again. I am assuming making a multi-document addon would be best to continue working this, so we work in modes and their brushes respectively. Do you think this is the correct method to do this?

**Comment by iyadahmed** [comment]
Yes it is best to make it multi file

But without duplicating code

**Comment by iyadahmed** [comment]
working on it currently

**Comment by iyadahmed** [comment]
progress
https://github.com/Bforartists/Bforartists/commit/fac775ce070ac0339c2070e6e5953e59114d71bf

**Comment by Draise14** [comment]
Ok, been taking a closer work, the stuff you did was incredible! But now I have no idea where to go with this, now it's beyond me! I notice now that you already have a good plan in place. Meaning it will be more dynamic? 

![image](https://user-images.githubusercontent.com/35357409/175447521-f68386c3-a343-4e14-ad79-d1460a78beed.png)

**Comment by iyadahmed** [comment]
Yep it currently has hardcoded things related to weight paint like "PAINT_WEIGHT", and so on, so I need to make these things variable, no worries working on it

**Comment by Draise14** [comment]
This task is getting big, so will close and open up smaller tasks per mode.
